test/xds/xds_server_rbac_test.go

```diff
@@ -605,6 +630,13 @@ func (s) TestRBACHTTPFilter(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			func() {
+				// TODO add check for these stats
+				lb := &loggerBuilder{
+					authzDecisionStat: map[bool]int{true: 2, false: 0},
+					lastEvent:         &audit.Event{},
```

Comment:
So we would just be changing around whether we are storing pointers/refs or just the actual object?

Reply:
No. It should still store pointers. But `cmp.Diff(nil, nil)` will be ok. So when neither `lastEvent` nor `test.eventContent` is populated the test should pass.

Reply:
Discussed offline, we can't use `nil` because for the pointer to correctly propagate from the logger to the logger builder we do `*s.lastEvent = *lastEvent`, setting the underlying content being pointed to.

If `s.lastEvent` is `nil`, this causes a segfault, so we have to set it to an `audit.Event{}`